treeedbgen-souffle/src/gen.rs

```diff
@@ -16,7 +16,10 @@ fn node_with_fields(
     w: &mut impl Write,
     node: &Node,
 ) -> Result<String, io::Error> {
-    let rel_name = &node.ty;
+    let rel_name = match node.ty.as_str() {
+    "true" | "false" => format!("{}_literal", node.ty),
+    _ => node.ty.clone(),
+    };
```

Comment:
We can likely avoid the `clone`:
```suggestion
    let rel_name = match node.ty.as_str() {
      "true" => "true_literal",
      "false" => "false_literal",
      _ => &node.ty,
    };
```